pkg/asset/agent/image/ignition.go

```diff
@@ -215,6 +215,7 @@ func (a *Ignition) Generate(dependencies asset.Parents) error {
 
 	agentEnabledServices := []string{
 		"agent.service",
+		"agent-interactive-console.service",
```

Comment:
Thanks for naming it `agent-<...>`, in the long term I think it'd be easier to rename all of our services with a known fixed prefix (for the sake of troubleshooting)